webview_panels/src/lib/altimate/main.js

```diff
@@ -40,12 +40,12 @@ var Sn = /* @__PURE__ */ ((e) => (
     typeof globalThis < "u"
       ? globalThis
       : typeof window < "u"
-      ? window
-      : typeof global < "u"
-      ? global
-      : typeof self < "u"
-      ? self
-      : {};
+        ? window
```

Comment:
most of the changes in this file are formatting changes and this is generated from altimate-components